src/embed.ts

```diff
@@ -225,7 +227,7 @@ export default async function embed(
         .attr('download', `visualization.${ext}`)
         .on('mousedown', function(this: HTMLLinkElement) {
           view
-            .toImageURL(ext)
+            .toImageURL(ext, scaleFactor)
```

Comment:
You can just pass opt.scaleFactor. it's okay to pass in undefined.

Reply:
you mean `toImageURL(ext, opt.scaleFactor)` without the default defined at line 103 ? Shall I also re-adjust the docs as well then? (docs mention the default of 1)

Reply:
The default is still 1 as that's what Vega uses but there is no need to redefine the default here.